packages/cli/test/helpers/stubs/delete.ts

```diff
@@ -0,0 +1,31 @@
+'use strict'
+import * as nock from 'nock'
+
+export function collaboratorsTeamApp(app: string, email: string) {
+  return nock('https://api.heroku.com:443', {
+    reqheaders: {Accept: 'application/vnd.heroku+json; version=3'},
+  })
+    .delete(`/teams/apps/${app}/collaborators/${email}`).reply(200, {})
+}
+
+export function collaboratorsPersonalApp(app: string, email: string) {
+  return nock('https://api.heroku.com:443', {})
+    .delete(`/apps/${app}/collaborators/${email}`).reply(200, {})
+}
+
```

Comment:
This is the only function used in tests for this command. Are the other functions used on other related commands, or is it again that there are tests that were never written for this command?

Reply:
I just migrated them all since they were easy to migrate to Typescript. I assume that they are used by other tests in the orgs-v5 commands.